configs/sandpack-contents/framer-motion-examples/chakra-factory.js

```diff
@@ -1,15 +1,14 @@
 module.exports = {
-  App: `import { Container, chakra } from '@chakra-ui/react';
+  App: `import { Container, chakra, shouldForwardProp } from '@chakra-ui/react';
 import { motion, isValidMotionProp } from 'framer-motion';
-  
+
 const ChakraBox = chakra(motion.div, {
   /**
-   * Allow motion props and the children prop to be forwarded.
-   * All other chakra props not matching the motion props will still be forwarded.

```

Comment:
I've removed this second part of the JSDocs description because I found it confusing (and I think that it was incorrect with the code before my change as non-motion props were not forwarded at all)